ajax_select/lookup_channel.py

```diff
@@ -97,7 +97,7 @@ def get_objects(self, ids):
         pk_type = self.model._meta.pk.to_python
         ids = [pk_type(pk) for pk in ids]
         things = self.model.objects.in_bulk(ids)
-        return [things[aid] for aid in ids if aid in things]
+        return things.values()
```

Comment:
I just tested again, this change is indeed not cessary. the type of ids is UUID and pk_type does not transform it. I checked with the mysql and sqlite backends (the latter previously did not work, but something must have changed in my setup).